android/app/src/main/java/com/simplecityapps/shuttle/ui/screens/library/genres/GenreListItem.kt

```diff
@@ -0,0 +1,107 @@
+package com.simplecityapps.shuttle.ui.screens.library.genres
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.pluralStringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.simplecityapps.shuttle.R
+import com.simplecityapps.shuttle.model.Genre
+import com.simplecityapps.shuttle.model.MediaProviderType
+import com.simplecityapps.shuttle.model.Playlist
+import com.simplecityapps.shuttle.persistence.GeneralPreferenceManager
+import com.simplecityapps.shuttle.sorting.PlaylistSongSortOrder
+import com.simplecityapps.shuttle.ui.screens.playlistmenu.PlaylistData
+import com.simplecityapps.shuttle.ui.theme.AppTheme
+
+@Composable
+fun GenreListItem(
+    genre: Genre,
+    playlists: List<Playlist>,
+    modifier: Modifier = Modifier,
+    onSelectGenre: (genre: Genre) -> Unit = {},
+    onPlayGenre: (Genre) -> Unit = {},
+    onAddToQueue: (Genre) -> Unit = {},
+    onPlayNext: (Genre) -> Unit = {},
+    onExclude: (Genre) -> Unit = {},
+    onAddToPlaylist: (playlist: Playlist, playlistData: PlaylistData) -> Unit = { _, _ -> },
+    onEditTags: (Genre) -> Unit = {},
+    onShowCreatePlaylistDialog: (genre: Genre) -> Unit = {}
+) {
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Column(
+            Modifier
+                .padding(start = 8.dp)
+                .weight(1f)
+                .clickable { onSelectGenre(genre) }
+        ) {
+            Text(
+                modifier = Modifier.fillMaxWidth(),
+                text = genre.name,
+                style = MaterialTheme.typography.bodyMedium,
+                color = MaterialTheme.colorScheme.onBackground
+            )
+            // Todo: Manually replacing "{count}" is not ideal. But, the Phrase library doesn't render correctly in Compose.
```

Comment:
Wouldn't the solution be to change the `{count}` in `strings_library.xml` by `%1$d`? We can't do so now because it'd break other uses of `songsPlural` but we could do so in the future, right?

Reply:
Yeah, that's right. I'm reluctant to change it because we'd have to revert the usage of the Phrase library which is all over the app at the moment